src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -225,8 +225,8 @@ public override INamedTypeSymbol CreateErrorTypeSymbol(INamespaceOrTypeSymbol co
                 options ?? s_defaultSubmissionOptions,
                 (syntaxTree != null) ? new[] { syntaxTree } : SpecializedCollections.EmptyEnumerable<SyntaxTree>(),
                 references,
-                previousScriptCompilation, 
```

Comment:
It looks like all changes in this file (src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs) are minor white-space changes, consider reverting.